app/models/student.rb

```diff
@@ -1,4 +1,18 @@
 class Student < User
   has_many :appointments
   has_many :checkin_records
+
+  validates :sid, presence: true
+  validate :check_is_student
+
+  def check_is_student
+    if is_student == false
+      raise "This user must be a student!!"
+    end
+  end
+
+  def checkin
```

Comment:
Right now I'm working on the check-in form that will accomplish what this method essentially does, so we may need to remove it later. The form that I have almost completed will autofill a user's information and create and check them in once they press submit on the form.